Assets/MixedRealityToolkit.Staging/License.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Are we leaving these files in the UnityAR folder as well? Seems redundant with these one level up now, and no other subfolder

Reply:
i can remove them